lib/fs-wrapper.js

```diff
@@ -78,10 +78,13 @@ function mkdirpSync(dir) {
 
 /**
  * @param {fs.PathLike} file
- * @param {fs.RmOptions} [options={recursive: true}]
  */
-async function remove(file, options) {
-  await fsp.rm(file, {recursive: true, ...options});
+async function remove(file) {
+  await fsp.rm(
+    file,
+    // Rimraf compatibility:
+    {recursive: true, force: true, maxRetries: 10}
```

Comment:
`rimraf` retries 10 times? Wow, that's surprising. I'm not sure I understand why it does that. I guess it's fine to keep...? There's not many downsides to it, but it's surprising that there is a need to retry this kind of operation.

Reply:
rimraf retries 10 times _with an exponential backoff_! We just use the default _linear backoff_, otherwise we might as well just be using rimraf.